app/api/check-auth/route.ts

```diff
@@ -1,6 +1,6 @@
 import { NextResponse } from 'next/server';
 import { getAuthTokenServer } from '@/utils/getAuthTokenServer';
-import { verifyJwtToken } from '../../../lib/server/auth'
+import { verifyJwtToken } from '@/lib/server/auth';
```

Comment:
Using `@` because the app's been configured to use that as root path, so just trying to be consistent